src/enums/nodeType.ts

```diff
@@ -1,10 +1,12 @@
 enum NodeType {
+    START = 'S',
     STOP = 'P',
     CROSSROAD = 'X',
     MUNICIPALITY_BORDER = '-',
     INVALID = 'invalid',
     DISABLED = 'E',
     TIME_ALIGNMENT = '1',
+    IS_NEIGHBOR = 'N',
```

Comment:
What does this mean?

Reply:
Means node is type can be neighbor. At the moment neighbor type is used when creating new routePaths, purple large nodes (neighbor nodes) have nodeType='N'

Reply:
This could be renamed as "IS_NEW_ROUTEPATH_NEIGHBOR" but i think the property name should not describe where it is being used

Reply:
Hmm okay, I was just a bit confused since a lot of this refers to how jore works. I guess this makes sense.